Examples/Temperature_Salinity_Diagram.ipynb

```diff
@@ -30,9 +30,11 @@
    "outputs": [],
```

Comment:
<p>This block doesn't work - the key it looks for (panant-01-zstar-v13) doesn't exist.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
cc @adele-morrison?